src/plugins/data_source_management/public/components/data_source_multi_selectable/data_source_filter_group.tsx

```diff
@@ -148,7 +153,14 @@ export const DataSourceFilterGroup: React.FC<DataSourceFilterGroupProps> = ({
               showIcons={true}
               style={itemStyle}
             >
-              {item.label}
+              <EuiFlexGroup alignItems="center">
```

Comment:
Looks like the default icon can be extracted into its own module since it will be shared among multiple components

Reply:
would do